EMS/admin-ui-bundle/src/Resources/views/bootstrap5/user/profile/edit.html.twig

```diff
@@ -0,0 +1,46 @@
+{% extends '@EMSAdminUI/bootstrap5/user/profile/show.html.twig' %}
+{%- trans_default_domain 'user' -%}
+
+{% block title %}{{ 'user.profile.edit.title'|trans }}{% endblock title %}
+
+{% block breadcrumb %}
+    {% set breadcrumb = {(path('emsco_user_profile_edit')): 'user.profile.edit.breadcrumb_title'|trans}|merge(breadcrumb)  %}
+    {{ parent() }}
+{% endblock breadcrumb %}
+
+{% block content %}
+    {{ form_start(form, { 'action': path(routes.USER_PROFILE_EDIT) }) }}
+    <div class="row">
+        <div class="col-md-6 offset-md-3">
+            <div class="card card-{{ theme_color }} card-outline">
+                <div class="card-body">
+                    {{ form_row(form.displayName) }}
+                    {{ form_row(form.email) }}
+                    {{ form_row(form.emailNotification) }}
+                    {{ form_row(form.current_password) }}
+
+                    {{ form_row(form.layoutBoxed) }}
+                    {{ form_row(form.sidebarMini) }}
+                    {{ form_row(form.sidebarCollapse) }}
+
+                    {% if form.userOptions.children|length > 0 %}
```

Comment:
Is the userOptions element always defined?

Reply:
Yes it's always defined, sometimes it's empty